pkg/rke/k8s_rke_system_images.go

```diff
@@ -9720,6 +9720,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.7.9
```

Comment:
super nit but I missed this earlier: this would be post v2.7.10 now that v2.7.10 has been released